pcsx2-qt/MainWindow.cpp

```diff
@@ -44,6 +44,8 @@
 #include "Settings/InterfaceSettingsWidget.h"
 #include "SettingWidgetBinder.h"
 #include "svnrev.h"
+#include <input-rec/NewInputRecordingDlg.h>
```

Comment:
```suggestion
#include "input-rec/NewInputRecordingDlg.h"
```

(and put it in alphabetical order)